Procfile

```diff
@@ -0,0 +1,4 @@
+# Run Rails & Webpack concurrently
+# Example file from webpack-rails gem
+rails: bundle exec rails server
```

Comment:
TBH I'm inclined to drop the foreman runner, it makes debugging with pry a PITA. since this is only for devel anyways, we could do with running the two servers separately.